pystructurizr/dsl.py

```diff
@@ -74,7 +74,7 @@ def dump_relationships(self, dumper: Dumper) -> None:
 
 
 class Person(Element):
-    def dump(self, dumper: Dumper) -> None:
+    def dump(self, dumper: Dumper, with_docs: bool = True) -> None:
```

Comment:
I dislike this, but because `Workspace` stores `Person`s and `SoftwareSystem`s as `List[Element]`, they both need to implement the same interface :grimacing: